### PR TITLE
Corrected Polish translation for "minute"

### LINKED ISSUE #173: Incorrect Polish translation
Thanks for great theme with good i18n support.  I've just spotted out a minor grammatical error in the Polish locale.
https://github.com/victoriadrake/hugo-theme-introduction/blob/8b1684b6c27c51bde5c4bb058e40e288089eb129/i18n/pl.toml#L19-L21
The word ["minut" is mispelled](https://dictionary.cambridge.org/spellcheck/polish-english/?q=minut), and it should be "minuta" and "minuty" in singular and plural forms respectively.  Creating a PR for that.

### PR DESCRIPTION
**Pull request by VincentTam**
# Motivation

1. To resolve #173.
2. To fxed the broken Polish translation reported in https://github.com/gohugoio/hugo/issues/3846.

# Additional context

1. [Unicode's plural rules](http://www.unicode.org/cldr/charts/28/supplemental/language_plural_rules.html).
2. https://en.wikipedia.org/wiki/Polish_morphology#Nouns
3. http://www.skwierzyna.net/polishgrammar.pdf

### CODE REVIEW

File: i18n/pl.toml [hunk 1]
```diff
@@ -17,8 +17,10 @@
 [nav_main]
     other = "główna"
 [blog_readingTime]
-    one = "{{ .Count }} minut"
-    other = "{{ .Count }} minut"
+    one = "{{ .Count }} minuta"
+    few = "{{ .Count }} minuty"
+    many = "{{ .Count }} minuty"
```
Comment (hanzei) [inline i18n/pl.toml]: Why do you need extra translations for `few` and `many` if they the same as `other`?

Comment (VincentTam) [reply i18n/pl.toml]: Please read the author's 2nd post in the linked GitHub issue https://github.com/gohugoio/hugo/issues/3846 and the 1st reference [Unicode's plural rules](http://www.unicode.org/cldr/charts/28/supplemental/language_plural_rules.html) for different languages (especially Polish).

Please compare the screenshots in my related [Hugo Discourse question](https://discourse.gohugo.io/t/plural-doesnt-work-for-brazillian-portuguese-and-polish/20329).

Comment (hanzei) [reply i18n/pl.toml]: I see. Would you please document, that this is a workaround and targets a bug in hugo.

Comment (VincentTam) [reply i18n/pl.toml]: OK for further explanation.  However, that's the expected behavior in Hugo.  Instead, this is a bug in this theme's template.

**Comment by VincentTam** [comment]
Explanation in response to @hanzei's comment:

The following code is incomplete.  To see this, pick any article whose `{{ .Count }}` is evaluated to a number that falls into the case of either `few` or `many`, say, `24`.

https://github.com/victoriadrake/hugo-theme-introduction/blob/8b1684b6c27c51bde5c4bb058e40e288089eb129/i18n/pl.toml#L19-L21

Hugo is written in Golang, which follows [Unicode's Language Plural Rules](http://www.unicode.org/cldr/charts/28/supplemental/language_plural_rules.html).  Here's the rule for Polish:

- `one`: i = 1
and v = 0
- `few`: v = 0 and
  i % 10 = 2..4 and
  i % 100 != 12..14
- `many`: either one of
    - v = 0 and  i != 1 and  i % 10 = 0..1
    - v = 0 and  i % 10 = 5..9
    - v = 0 and  i % 100 = 12..14
- `other`: remaining cases (decimals)

Note that the above quoted code block covered none of the whole numbers greater than one.  My PR aims at fixing this grammatical issue, in order to provide a grammatically correct template for the Polish language.

**Comment by victoriadrake** [comment]
For clarification, @VincentTam do you speak Polish fluently? If not, I would like a review from someone who does just to ensure this comes out making sense.

**Comment by VincentTam** [comment]
> For clarification, @VincentTam do you speak Polish fluently? If not, I would like a review from someone who does just to ensure this comes out making sense.

No, but that doesn't stop me from finding the solution.

**Pull request merged**